doc/getting_started/index.rst

```diff
@@ -159,6 +159,14 @@ Set Up a Toolchain
    setting :envvar:`ZEPHYR_TOOLCHAIN_VARIANT` appropriately for a non-SDK
    toolchain.
 
+   As another way to skip setting up a toolchain, you may be able for
+   some configurations (e.g.: non-MCU x86 and x86_64 boards on Linux) to
+   re-use the development tools provided by your operating system that
+   `you already installed in the first step`__. To try this option use:
+   ``ZEPHYR_TOOLCHAIN_VARIANT=host``
+
+__ host_setup_
```

Comment:
delete this
```suggestion

```

Reply:
This is the double underscore link you think is missing and it Just Works for me... this is regular .rst syntax, nothing fancy?!

Reply:
So this anonymous link was working but @dbkinder showed me a better <> way, Done.

Reply:
Ah, OK.  We've not been using anonymous links, but they're OK as is.  My preference would be to use the alternate name option for links:  ``  `alternate text <link_target_name>`_ `` to override the use of the link target title as the display text (instead of using anonymous links).